mvc/controllers.js

```diff
@@ -37,10 +38,29 @@ exports.getArticleByIdController = (req, res, next) => {
 
 exports.getCommentsByArticleIdController = (req, res, next) => {
     const { article_id } = req.params
-    const promises = [getCommentsByArticleIdModel(article_id), checkCategoryExists(article_id)]
+    const promises = [
+        getCommentsByArticleIdModel(article_id),
+        checkCategoryExists(article_id),
```

Comment:
This function may need a more semantic name if it is reusable. For getting comments I don't understand why I'd be looking if a category exists.